examples/java/BUILD

```diff
@@ -52,9 +52,10 @@ java_fuzz_test(
     tags = [
         "no-oss-fuzz",
     ],
-    deps = [
-        ":native",
-    ],
+    deps = select({
+        "@platforms//os:macos": [":libnative.dylib"],
+        "//conditions:default": [":native"],
+    }),
```

Comment:
Good point. Since this is a workaround for a bug in Bazel that might get fixed in Bazel 5, I will wait until that has been decided and update the comment accordingly.

Reply:
@stefanbucur I added more comments, could you take another look? The bug will not be fixed for Bazel 5, so I tried to explain the workaround in more detail.

Reply:
Thank you, this is clear!